### PR TITLE
Add v2tofhir custom service template

### PR DESCRIPTION
**Pull request by sameeragunarathne**

### CODE REVIEW

File: r4/v2-to-fhir-custom-mapper-service/service.bal [hunk 1]
```diff
@@ -0,0 +1,135 @@
+// Copyright (c) 2024, WSO2 LLC. (http://www.wso2.com).
```
Comment (nirmal070125) [inline r4/v2-to-fhir-custom-mapper-service/service.bal]: Move to a sub folder `data-mappers`.

File: r4/v2-to-fhir-custom-mapper-service/service.bal [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+import ballerina/http;
+import ballerinax/health.hl7v2commons;
+
+// You can implement custom mapping logic within resource functions for each segment in the HL7v2 message to 
+// convert it to a FHIR resource.
+service /v2tofhir on new http:Listener(9091) {
```
Comment (nirmal070125) [inline r4/v2-to-fhir-custom-mapper-service/service.bal]: Add `custom` part to the context.

File: r4/v2-to-fhir-custom-mapper-service/service.bal (hunk 1 above)
Comment (sameeragunarathne) [reply r4/v2-to-fhir-custom-mapper-service/service.bal]: fixed with 4ce07f9f38de973d0024c750e49923240e35fc3d

File: r4/v2-to-fhir-custom-mapper-service/service.bal (hunk 2 above)
Comment (sameeragunarathne) [reply r4/v2-to-fhir-custom-mapper-service/service.bal]: Fixed with 4ce07f9f38de973d0024c750e49923240e35fc3d

**Review by nirmal070125 — APPROVED** [review]

**Pull request merged**